common/src/main/java/com/wynntils/features/user/overlays/CustomBarsOverlayFeature.java

```diff
@@ -301,22 +294,14 @@ public ManaBankBarOverlay() {
         public void render(PoseStack poseStack, float partialTicks, Window window) {
             if (!WynnUtils.onWorld()) return;
 
-            LerpingBossEvent bankEvent = null;
-            String manaBankPercent = "";
-            String manaBankMaxPercent = "";
-            for (LerpingBossEvent event :
-                    McUtils.mc().gui.getBossOverlay().events.values()) {
-                Matcher matcher = MANA_BANK_PATTERN.matcher(ComponentUtils.getCoded(event.getName()));
-                if (matcher.matches()) {
-                    bankEvent = event;
-                    manaBankPercent = matcher.group(1);
-                    manaBankMaxPercent = matcher.group(2);
-                    break;
-                }
-            }
+            Pair<LerpingBossEvent, Matcher> matcherPair = WynnBossBarUtils.getLerpingBossEvent(MANA_BANK_PATTERN);
+            LerpingBossEvent bankEvent = matcherPair.a;
 
             if (bankEvent == null) return;
 
+            String manaBankPercent = matcherPair.b.group(1);
```

Comment:
Can you like make a public method in WynnBossBarUtils like `getManaBank` and `getBloodPool(Percent)`? And move the patterns over there. For efficiency, let `getManaBank` return a Pair of strings. (From an API point of view, two methods returning the current and max value would have been nicest, but that'd require doubling the parsing.